### PR TITLE
Summernote as TextArea and addition of BootstrapResourceLink

### PR DESCRIPTION
**Pull request by ymlaine**
Hi, 

I'm still learning git and github, so I finally though that it's better to commit on the master (instead of wicket-7.x branch, am I right ?) and I merged 4 commits in one. So this is a new pull request that replace the yesterday one.

I also added a new BootstrapResourceLink (based on BootstrapLink) that I did in the past, is it ok to put these 2 commits in one pull request ? 

regards,
Yves-Marie

### CODE REVIEW

**Comment by martin-g** [comment]
> I also added a new BootstrapResourceLink (based on BootstrapLink) that I did in the past, is it ok to put these 2 commits in one pull request ?

No :-)
Please create separate PR for different contributions.

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapResourceLink.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+import de.agilecoders.wicket.core.markup.html.bootstrap.button.Buttons.Type;
+
+/**
+ * 
+ * Same as default {@link ResourceLink} but based on {@link BootstrapLink} which is styled by bootstrap. 
+ * 
+ * @author ymlaine
+ *
+ */
+public class BootstrapResourceLink extends BootstrapLink<Void> implements IResourceListener {
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapResourceLink.java]: I think there is no need of such component in the library.
You can just use: `new ResourceLink(...).add(new ButtonBehavior(...))`.
Copying big chunks of code from Wicket to the library begs for problems in the maintainance in the future

**Comment by martin-g** [comment]
Thanks for the contribution!
The code indeed looks much simpler now!

I've removed BootstrapResourceLink.

**Comment by klopfdreh** [comment]
Thanks from me, too! 

Good to see that the editor is going to be used / improved furtheron! :-)

**Pull request merged**